other/install_2-1_mysql.sql

```diff
@@ -1179,9 +1179,9 @@ CREATE TABLE {$db_prefix}user_likes (
 CREATE TABLE {$db_prefix}mentions (
   content_id INT DEFAULT '0',
   content_type VARCHAR(10) DEFAULT '',
-  id_mentioned INT DEFAULT 0,
-  id_member INT(10) UNSIGNED NOT NULL DEFAULT 0,
-  `time` INT NOT NULL DEFAULT 0,
+  id_mentioned INT DEFAULT '0',
```

Comment:
I'm not sure why we would ever use single quotes around integer values like that. It's a good idea to always pass them as INTs rather than strings. For now you can just fix it for this PR and we can do a separate one to clean up the rest.